includes/admin/discounts/add-discount.php

```diff
@@ -70,8 +70,8 @@
 							<input type="text" required="required" class="edd-price-field" id="edd-amount" name="amount" value="" placeholder="<?php esc_html_e( '10.00', 'easy-digital-downloads' ); ?>"/>
 							<label for="edd-amount-type" class="screen-reader-text"><?php esc_html_e( 'Amount Type', 'easy-digital-downloads' ); ?></label>
 							<select name="amount_type" id="edd-amount-type">
-								<option value="percent"><?php esc_html_e( '%', 'easy-digital-downloads' ); ?></option>
-								<option value="flat"><?php esc_html_e( '$', 'easy-digital-downloads' ); ?></option>
+								<option value="percent">%</option>
+								<option value="flat"><?php echo esc_html( edd_currency_filter( '' ) ); ?></option>
```

Comment:
What about using `edd_currency_symbol()` instead of `edd_currency_filter()` ?

I understand `edd_currency_filter()` does actually do exactly what we want here, but the original intention is for formatting a final amount with a price, even if it supports an empty string as a price.

Reply:
`edd_currency_filter()` is used so much in the codebase I'm not even sure I knew `edd_currency_symbol()` existed. 😬

`edd_currency_symbol()` makes sense!

Reply:
I'll swap it--I looked in the `metabox.php` file to see how the currency symbol was added there and went off of that.